tests/neg/i19334.scala

```diff
@@ -0,0 +1,6 @@
+
+def foo[T](f: T): T = ???
+
+@main def main = foo:
+  def f() = ()
+  f(_) // error was OOM formatting TypeVar(TypeParamRef(T)) when offering explanations
```

Comment:
it might be worth adding a `tests/neg/i19334.check` file for this one.

Reply:
footnote I had one locally. I'm unaccustomed to the dotty-local customs around check files.